packages/ui/src/components/Tabs.svelte

```diff
@@ -57,6 +57,7 @@
     flex-shrink: 0;
     flex-wrap: nowrap;
     margin-bottom: 0.5rem;
+    padding: 0 1.5rem;
```

Comment:
I not look good in Documents and other places.

<img width="833" alt="image" src="https://user-images.githubusercontent.com/477235/218508669-8507bd92-7953-4de8-9793-6c497483f6fc.png">